addons/im_livechat/security/ir.model.access.csv

```diff
@@ -18,3 +18,4 @@ access_chatbot_script_step_user,chatbot.script.step.user,model_chatbot_script_st
 access_chatbot_script_answer,chatbot.script.answer,model_chatbot_script_answer,,0,0,0,0
 access_chatbot_script_answer_user,chatbot.script.answer.user,model_chatbot_script_answer,im_livechat_group_manager,1,1,1,1
 access_chatbot_message_user,chatbot.script.user,model_chatbot_message,im_livechat_group_manager,1,1,1,1
+im_livechat.access_discuss_channel_member_history,access_discuss_channel_member_history,im_livechat.model_discuss_channel_member_history,base.group_user,0,0,0,0
```

Comment:
Livechat manager should probably have access to the history.

Reply:
It's not really a question, I'm sure I need to be able to access the history on the prod to help debug when issues are reported.